editor/filesystem_dock.cpp

```diff
@@ -1296,6 +1296,13 @@ void FileSystemDock::_fs_changed() {
 		_update_file_list(true);
 	}
 
+	if (!select_after_scan.is_empty()) {
+		navigate_to_path(select_after_scan);
```

Comment:
Looks to me that we can use the private method directly here, `_navigate_to_path`. Seems to be what we call consistently throughout this class and the public one doesn't do anything useful to this use case.